apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,245 @@
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.webp',
+    previewJson: articleLayoutPreview,
+  },
+] as const
+
+const dataAttr = (value: unknown) => (!!value ? true : undefined) // Keeping this here for now, might shift out to a utils file when necessary.
+
+// TODO: Make this headless by getting the LAYOUT_DATA from the schema. Find somewhere in the schema for layoutDescription & image(fetch this too or generate it)
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  const [selectedLayout, setSelectedLayout] = useState<LayoutDataType>(
+    LAYOUT_DATA[0]!,
+  )
+
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100%">
+        <VStack p="1.5rem 2rem" w="full" gap="0.5rem" alignItems="flex-start">
+          <Link variant="standalone" href="/" gap="0.25rem" size="sm" p="0">
+            <Icon as={BiLeftArrowAlt} boxSize="1.25rem" />
+            Back to all pages
+          </Link>
+          <HStack p="0" justifyContent="space-between" w="100%">
+            <Text textStyle="h4" color="base.content.default">
+              {`Choose a layout for "${props.pageName}"`}
+            </Text>
+            <Button size="xs">
+              Start with this layout
+              <BiRightArrowAlt fontSize="1.5rem" />
+            </Button>
+          </HStack>
+        </VStack>
+        <Grid
+          w="100%"
+          h="calc(100% - 7rem)"
+          templateColumns="repeat(4, 1fr)"
+          gap={0}
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((layoutEntry) => {
+                return (
+                  <VStack
+                    key={layoutEntry.layoutTypename}
+                    p="0"
+                    gap="0"
+                    onClick={() => {
+                      setSelectedLayout(layoutEntry)
+                    }}
+                    role="group"
+                    as="button"
+                  >
+                    <Box
+                      borderRadius="8px"
+                      border="2px"
+                      p="1.5rem 1.5rem 0 1.5rem"
+                      mb="1.25rem"
+                      position="relative"
+                      borderColor="base.divider.medium"
+                      bgColor="interaction.muted.main.hover"
+                      data-active={dataAttr(
+                        selectedLayout.layoutTypename ===
+                          layoutEntry.layoutTypename,
+                      )} // have to use null so that React will not parse this jsx attribute. having the [data-active] selection attribute will trigger _active.
```

Comment:
nit: unnecessary comment

```suggestion
                      )}
```